podcasts/End of Year/Stories/TopCategoriesStory/TopListenedCategoriesStory.swift

```diff
@@ -8,47 +8,77 @@ struct TopListenedCategoriesStory: ShareableStory {
 
     let listenedCategories: [ListenedCategory]
 
-    let contrastColor: CategoriesContrastingColors
-
     init(listenedCategories: [ListenedCategory]) {
         self.listenedCategories = listenedCategories
-        self.contrastColor = CategoriesContrastingColors(podcast: listenedCategories.reversed()[0].mostListenedPodcast)
     }
 
     var body: some View {
         GeometryReader { geometry in
             PodcastCoverContainer(geometry: geometry) {
-                let headerSpacing = geometry.size.height * 0.054
+                StoryLabelContainer(geometry: geometry) {
+                    let mostListenedCategory = listenedCategories.first?.categoryTitle.localized ?? ""
+                    let listenedTime = listenedCategories.first?.totalPlayedTime.storyTimeDescription ?? ""
+                    let listenedEpisodes = "\(listenedCategories.first?.numberOfEpisodes ?? 0)"
+                    StoryLabel(L10n.eoyStoryTopCategoriesTitle(mostListenedCategory), for: .title, geometry: geometry)
+                    StoryLabel(L10n.eoyStoryTopCategoriesSubtitle(listenedEpisodes, listenedTime), for: .subtitle, color: Color(hex: "8F97A4"), geometry: geometry)
+                }
 
-                StoryLabel(L10n.eoyStoryTopCategories, for: .title2)
-                    .opacity(0.8)
+                let headerSpacing = geometry.size.height * 0.054
 
-                HStack(alignment: .bottom, spacing: 25) {
-                    ForEach([1, 0, 2], id: \.self) {
-                        pillar($0, size: geometry.size)
+                VStack(alignment: .leading, spacing: geometry.size.height * 0.03) {
+                    ForEach(0...min(listenedCategories.count, 3), id: \.self) { index in
+                        category(row: index, geometry: geometry)
                     }
-                }.padding(.top, headerSpacing)
-            }.background(DynamicBackgroundView(backgroundColor: contrastColor.backgroundColor, foregroundColor: contrastColor.foregroundColor))
+                }
+                .padding([.leading, .trailing], 35)
+                .padding(.top, headerSpacing)
+            }
+            .background(
+                ZStack(alignment: .top) {
+                    Color.black
+
+                    StoryGradient()
+                    .offset(x: -geometry.size.width * 0.4, y: -geometry.size.height * 0.22)
+                    .clipped()
+                }
+                .ignoresSafeArea()
+            )
```

Comment:
This will fix the clipping and the extra space at the bottom:

```suggestion
            .background(
                ZStack(alignment: .top) {
                    Color.black

                    StoryGradient()
                    .offset(x: -geometry.size.width * 0.4, y: -geometry.size.height * 0.22)
                }
            )
        }
        .clipped()
        .ignoresSafeArea()
```

Reply:
The text is not readable on smaller screens because the gradient clashes with it. 

Question: Is there a reason the StoryGradient is fixed at 510x510? I feel like scaling the size using the screen width would produce a more consistent result.

Example:

### Before:

| iPod Touch | iPhone 14 Pro | iPhone 15 Pro Max |
|:---:|:---:|:---:|
|<img src="https://github.com/Automattic/pocket-casts-ios/assets/793774/5c90e26a-ef58-46b3-ba83-aba8343b1ba7" height="320" />|<img src="https://github.com/Automattic/pocket-casts-ios/assets/793774/8b4b40f8-da09-4ba2-bc13-c343634033ce" height="320" />|<img src="https://github.com/Automattic/pocket-casts-ios/assets/793774/f6f61647-f92a-4aa2-b4e1-c5b2fccffd46" height="320" />|

### After using 130% of device width:

| iPod Touch | iPhone 14 Pro | iPhone 15 Pro Max |
|:---:|:---:|:---:|
|<img src="https://github.com/Automattic/pocket-casts-ios/assets/793774/15b68f3f-f732-4c11-addb-53c4401dc425" height="320" />|<img src="https://github.com/Automattic/pocket-casts-ios/assets/793774/b08f38cb-4c28-4ed2-9c66-6e6d96226b5b" height="320" />|<img src="https://github.com/Automattic/pocket-casts-ios/assets/793774/a6e7dfde-5ab2-47a7-9381-cfb22e4355b7" height="320" />|

Reply:
> Question: Is there a reason the StoryGradient is fixed at 510x510? I feel like scaling the size using the screen width would produce a more consistent result.

Nope! This should also be relative to the screen height. I documented it here: #1142